research/src/components/global.css

```diff
@@ -1,3 +1,67 @@
+/*       Header     */
+.header-menu-btn[type='button'] {
+  display: none;
+  padding: 0.25em 0.5em;
+  appearance: none;
+  border: 0;
+  background: none;
+  color: white;
+  cursor: pointer;
+}
+
+.header-menu-btn[type='button']:hover {
+  background-color: #666;
+}
+
+@media (max-width: 640px) {
```

Comment:
I chose 640px semi-randomly (I just used the various device emulations devtools has to see what looked good on that sampling of devices). I imagine people here have done much more research on this so feedback here would be appreciated!